### ISSUE TITLE
Adding background.js to extension per instructions throws an error trying to load

### ISSUE THREAD

**Issue by elgreg** [GoogleChrome/developer.chrome.com#322]
**Describe the bug**

Following the instructions at https://developer.chrome.com/docs/extensions/mv3/getstarted and trying to load the extension produces an error:

> The "background.scripts" key cannot be used with manifest_version 3. Use the "background.service_worker" key instead. Could not load manifest.

**To Reproduce**
1. Create the manifest.json from the getstarted (see additional context below)
2. Try to load it in Chrome Developer Mode
3. See error

**Expected behavior**

Should load extension? Need to use `serviceworker` key?

**Screenshots**

<img width="531" alt="Extensions 2021-01-26 17-16-05" src="https://user-images.githubusercontent.com/40772/105913094-5ce66d80-5ffa-11eb-80fa-4e0f78763a6f.png">

**Additional context**

Sample manifest.json

```
{
	"name": "Thing Doer",
	"version": "1.0",
	"description": "Doer of things!",
	"manifest_version": 3,
	"background": {
		"scripts": [
			"background.js"
		],
		"persistent": false
	},
	"permissions": [
		"contextMenus",
		"storage"
	]
}
```

**Comment by robdodson** [comment]
@dotproto Is this still an issue now that the guide has been updated?

**Comment by dotproto** [comment]
I believe this was one of the many issues addressed by PR #317.

**Comment by abhishekyadav1379** [comment]
{
	"name": "Thing Doer",
	"version": "1.0",
	"description": "Doer of things!",
	"manifest_version": 2,
	"background": {
		"scripts": [
			"background.js"
		]
	},
	"permissions": [
		"contextMenus",
		"storage"
	]
}

it might be work
